src/index.ts

```diff
@@ -158,6 +163,17 @@ if (argv) {
       .catch((e) => {
         throw new Error(e);
       });
+  } else if (argv._[0] === 'clear') {
+    let config = new OssIndexServerConfig();
+    config.getConfigFromFile();
+    console.log('Cache location:', config.getCacheLocation());
+    config.clearCache()
```

Comment:
No strong opinion here, but you could do `try  { await config.clearCache(); } catch (e)  { doSomethingWithError(e) }` if you don't want to do the whole promise thing.

Reply:
Haha so funny that you mentioned this -- I was deciding upon doing it that but but in an effort to keep things consistent i opted to use then like in (lines 165-169)

Reply:
Yeah buyers choice. I prefer the try catch await stuff because it is how I'm used to programming. If then etc... works better for you, do it! As long as the file is consistent than I'm good (we should probably not mix paradigms as much as we can, except for when forced, cough cough `readinstalled`)